opentracing-noop/src/main/java/io/opentracing/noop/NoopTracer.java

```diff
@@ -44,7 +55,7 @@ public ActiveSpan activeSpan() {
 
     @Override
     public ActiveSpan makeActive(Span span) {
```

Comment:
There is also `activeSpan()`. It currently returns `null` I'm wondering whether it should return something.